Fix pdf issue by removing it - part 2 (#359)
zeus-health/ctw-component-library: .changeset/tricky-eyes-refuse.md

---
"@zus-health/ctw-component-library": patch
---

> Remove pdf libraries to fix up an import issue.

Reviewer: ```suggestion
Remove `react-pdf-viewer` and `react-json-view` to fix server rendering issues. 
```
